_includes/utilities/js/helpers/wrapStringPhrases.js

```diff
@@ -1,15 +1,26 @@
-export default function wrapStringPhrases(string, phrases, templateFn) {
+export default function wrapStringPhrases(string, phrasesArray, templateFn, options = {}) {
+  // Removes phrases that have less charafter than 'minChars' if specified
+  const removeIfNotEnoughChars = item => item.length > options.minChars;
+  const returnAsIs = item => item;
+  const phrases = phrasesArray.filter(options.minChars ? removeIfNotEnoughChars : returnAsIs);
+
+  // Removes html from results if 'excludeHtml' is specified
+  const excludeHtmlConfig = options && options.excludeHtml;
+  const regexWrap = excludeHtmlConfig ? item => `(?!<[^<]*)${item}(?![^<>]*>)` : item => item;
```

Comment:
there's no reason for us to implement removing html from strings - much smarter people have done it before us.

Reply:
unless I'm missing something?

Reply:
@jbothma It does not remove the HTML from the string, since we need the HTML for the actual output on the UI, it just tells the regex to not match anything that is inside `<>` brackets.

Reply:
what would result in that? where would additional < and > come from?

Reply:
what's the regex used for?

Reply:
It's used both for finding tooltips and wrapping the tooltip component around them on the site. It is also used in highlighting words/phrases that correspond to search terms in search results.

There are instances where markup needs to be searched, for example the following example from FAQs:

```
---
title: The data I need is not on the site, will you be adding additional data and resources?
---

The development of vulekamali is ongoing and additional data and resources are still being added to the site. [Read more about the project timelines](/about) to learn about what types of datasets will be added, and when.

Please tell us about the data you’re looking for by emailing us at feedback@vulekamali.gov.za.
```